tests/query_context_tests.py

```diff
@@ -73,13 +74,41 @@ def test_schema_deserialization(self):
                 self.assertEqual(post_proc["operation"], payload_post_proc["operation"])
                 self.assertEqual(post_proc["options"], payload_post_proc["options"])
 
-    def test_cache_key_changes_when_datasource_is_updated(self):
+    def test_cache(self):
+        table_name = "birth_names"
+        table = self.get_table_by_name(table_name)
+        payload = get_query_context(table.name, table.id, table.type)
+        payload["force"] = True
+
+        query_context = ChartDataQueryContextSchema().load(payload)
+        query_object = query_context.queries[0]
+        query_cache_key = query_context.query_cache_key(query_object)
+
+        response = query_context.get_payload(cache_query_context=True)
+        cache_key = response["cache_key"]
+        assert cache_key
+
+        cached = cache_manager.cache.get(cache_key)
+        assert cached
```

Comment:
nit: should we `assert cached is not None` here to highlight that we're not expecting a truthy value but something that isn't `None`?